src/core/moves/compound/RateAgeBetaShift.cpp

```diff
@@ -502,43 +502,44 @@ void RateAgeBetaShift::printSummary(std::ostream &o, bool current_period) const
 
 void RateAgeBetaShift::reject( void )
 {
-    
-    // undo the proposal
-    tree->getValue().getNode( stored_node->getIndex() ).setAge( stored_age );
-    
-    // undo the rates
-    size_t node_idx = stored_node->getIndex();
-    if ( rates == NULL )
-    {
-        rates_vec[node_idx]->setValue(new double(stored_rates[node_idx]));
-    }
-    else
-    {
-        rates->getValue()[ node_idx ] = stored_rates[node_idx];
-        rates->touch();
-    }
-    for (size_t i = 0; i < stored_node->getNumberOfChildren(); i++)
+    if (stored_node != NULL)
```

Comment:
I think you could replace this with ` if (stored_node == nullptr) return;`.  Then you wouldn't need to indent everything.